supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -303,6 +303,17 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*:
 
+[discrete]
+[[cidr]]
+==== image:images/yes.png[yes] CIDR (noun)
+*Description*: Classless Inter-Domain Routing ("CIDR") is a method to efficiently allocate IP addresses and for IP routing. CIDR replaces the classful network addressing architecture. In CIDR notation, IP addresses contain a suffix that represents the number of bits of the prefix. Expand on first usage, and write it as shown: "Classless Inter-domain Routing".
```

Comment:
Since you changed it earlier in the description: 

```suggestion
*Description*: Classless Inter-Domain Routing (CIDR) is a method to efficiently allocate IP addresses and for IP routing. CIDR replaces the classful network addressing architecture. In CIDR notation, IP addresses contain a suffix that represents the number of bits of the prefix. Expand on first usage, and write it as shown: "Classless Inter-Domain Routing".
```
Also, should the example in the last sentence include (CIDR) in there?

Reply:
Argh, I missed this one. Fixed.